pyfar/plot/line.py

```diff
@@ -351,23 +351,24 @@ def freq_group_delay(signal, log_prefix=20, log_reference=1, ax=None,
 
     with plt.style.context(plotstyle(style)):
         ax = _line._freq_group_delay(signal, log_prefix, log_reference,
-                                     **kwargs)
+                                     ax, **kwargs)
 
     plt.tight_layout()
     Interaction('LineXLog', ax[0], signal, style, **kwargs)
 
     return ax
 
 
-def summary(signal, ax=None, style='light', **kwargs):
+def multi(signal, plots, ax=None, style='light', **kwargs):
```

Comment:
Multi is a very arbitrary function name. Could we come up with something more explicit?

Reply:
I agree, but do not have the best ideas. I came up with:
- lines
- multi_line

Reply:
Could this function be extended to non-line plots? What about a name with something like subplots?